subprojects/code-quality/src/main/groovy/org/gradle/api/plugins/quality/PmdPlugin.java

```diff
@@ -136,6 +136,12 @@ public Boolean call() {
                 return extension.isIgnoreFailures();
             }
         });
+        taskMapping.map("maxFailures", new Callable<Integer>() {
```

Comment:
These have been converted to lambdas on master

Reply:
Thanks, updated in the rebase.